### PR TITLE
fix(learners): fix bring items from ant design to NextUI

### PR DESCRIPTION
**Pull request by GarrixA**
# This fix is here for:
- fixing design components from antd to NextUi

## How to test this ?
- check in learners -> page.tsx if no components from antd still there

### CODE REVIEW

File: components/QuantitiesEmployerSeek.tsx [hunk 1]
```diff
@@ -7,35 +8,37 @@ const QuantitiesEmployerSeek = () => {
...
-					<li className="text-lg">
+					<li className="text-lg lg:text-xl xl:text-2xl">
 						<b>Computer Literacy: </b>Digital learning modules and tools enhance
 						students' proficiency in technology.
 					</li>
-					<li className="text-lg">
+					<li className="text-lg lg:text-xl xl:text-2xl">
 						<b>Teamwork: </b>Collaborative projects and community interactions
 						build teamwork skills.
 					</li>
```
Comment (fabmukunzi) [inline components/QuantitiesEmployerSeek.tsx]: Let's keep fonts not too large

File: app/learners/page.tsx [hunk 2]
```diff
@@ -1,125 +1,145 @@
...
-      </div>
-    </>
-  );
+	return (
+		<>
+			<div>
+				<div className="flex justify-between mt-10 lg:flex-row flex-col space-y-10 lg:space-y-0">
+					<div className="lg:w-1/2 flex flex-col justify-between space-y-3 sm:space-y-10 lg:space-y-3">
+						<h3 className="text-black text-lg">Posinnove for learners</h3>
+						<h1 className="text-primary text-xl sm:text-xl lg:text-2xl xl:text-4xl font-semibold pt-8 lg:pt-16 ">
```
Comment (fabmukunzi) [inline app/learners/page.tsx]: Let's add primary color and bold this

**Review by fabmukunzi — CHANGES_REQUESTED** [review]

File: components/QuantitiesEmployerSeek.tsx (hunk 1 above)
Comment (GarrixA) [reply components/QuantitiesEmployerSeek.tsx]: Done

File: app/learners/page.tsx (hunk 2 above)
Comment (GarrixA) [reply app/learners/page.tsx]: Done

**Pull request merged**